frontend/public/actions/features.ts

```diff
@@ -207,11 +209,23 @@ const detectCanCreateProject = (dispatch) =>
     },
   );
 
+const fetchUser = async (): Promise<SelfSubjectReviewKind> => {
+  try {
+    const response = await coFetch('/apis/authentication.k8s.io/v1/selfsubjectreviews', {
```

Comment:
This is the key part of the change.  This API has been available in kube for a while and it's a complete replacement for the users/~